rudder-sdk-core/src/main/java/com/rudderlabs/android/sdk/core/Constants.java

```diff
@@ -4,6 +4,8 @@
  * Default value holder class
  * */
 class Constants {
+    // how often config should be fetched from the server (in hours) (2 hrs by default)
+    static final int CONFIG_REFRESH_INTERVAL = 2;
```

Comment:
Can we also add MAX_CONFIG_REFRESH_INTERVAL and MIN_CONFIG_REFRESH_INTERVAL here?

It makes it easy for someone who is changing this to realize these are bounded by two more variables without going into the implementation.